### PR TITLE
NEW ARTICLE: Add an Office 365 license 

### PR DESCRIPTION
**Pull request by DonNateR**
- From PR #2138

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-8fba8d832f/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-092e4dc7d7/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-28dcf25789/how-to/)

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits attached. 

[Add an Office 365 License_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1132252/Add.an.Office.365.License_editsKH.docx)
[Add an Office 365 License_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1132253/Add.an.Office.365.License_editsKH.pdf)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b4baa34294/how-to/)

File: content/office-365/add-an-office-365-license.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+permalink: add-an-office-365-license/
+audit_date:
```
Comment (stephamon) [inline content/office-365/add-an-office-365-license.md]: add today's date for the audit_date

File: content/office-365/add-an-office-365-license.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+### Add an Office 365 license
+
+1. Log in to the [Cloud Office Control Panel](https://cp.rackspace.com/) by using your Rackspace Cloud Office admin ID and password.
+2. In the Microsoft Office 365 section, click **Manage Office 365**.
+
+   <img src="{% asset_path office-365/add-an-office-365-license/add-office-365-sub-1.png %}" />
+
+3. If the "FIRST TIME SETUP", appears above the **Microsoft Office 365** title, perform the steps in [Set up Office 365 - Create a tenant ID](/how-to/office-365/set-up-office-365#create-a-tenant-id), and then return to this article.
```
Comment (stephamon) [inline content/office-365/add-an-office-365-license.md]: If "FIRST TIME SETUP" appears...

File: content/office-365/add-an-office-365-license.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+   If the "FIRST TIME SETUP" does not appear (because you have an existing tenant), click **Add Product** to add a new product subscription.
```
Comment (stephamon) [inline content/office-365/add-an-office-365-license.md]: If "FIRST TIME SETUP" does not appear...

**Comment by stephamon** [comment]
👍

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-a77abe5f7d/how-to/). :bow:

**Pull request merged**